src/layouts/Sidebar.jsx

```diff
@@ -8,127 +8,214 @@ import { useAuth } from "../context/AuthContext";
+                </a>
+                {isModuleActive && (
+                  <ul className="sub-links">
+                    <li
+                      className={`sub-link ${
+                        isSubActive.task ? "sub-active" : ""
+                      }`}
+                      onClick={() => {
+                        handleSubLinkClick("task");
+                      }}
+                    >
+                      <a
+                        href={`/batch/${batchId}/module/task`}
+                        onClick={() => isModuleActive(true)}
+                      >
+                        Task
+                      </a>
+                    </li>
+                    <li
+                      className={`sub-link ${
+                        isSubActive.assessment ? "sub-active" : ""
+                      }`}
+                      onClick={() => handleSubLinkClick("assessment")}
+                    >
+                      <a
+                        href={`/batch/${batchId}/module/assessment`}
+                        onClick={() => isModuleActive(true)}
+                      >
+                        Assessment
+                      </a>
+                    </li>
+                    <li
+                      className={`sub-link ${
+                        isSubActive.quiz ? "sub-active" : ""
+                      }`}
+                      onClick={() => handleSubLinkClick("quiz")}
+                    >
+                      <a href="#" onClick={() => isModuleActive(true)}>
+                        Quiz
+                      </a>
+                    </li>
+                  </ul>
+                )}
+              </li>
+              <li
+                className={`main-link ${
+                  isActive.settings ? "main-active" : ""
                 }`}
-                onClick={() => setActive(menu.id)}
               >
-                <div className="applicants flex">
-                  <span className="material-symbols-outlined">view_list</span>
-                  <p>{menu.label}</p>
-                </div>
-              </div>
-            </Tooltip>
-          </Link>
-        ))}
+                <a
+                  href="#"
+                  className="flex"
+                  onClick={() => handleMainLinkClick("settings")}
+                >
+                  <img src="/icons/application.svg" alt="home icon" />
+                  <span>Settings</span>
+                </a>
+              </li>
+            </>
+          )}
+        </ul>
       </div>
 
-      {!isDashboardPage && (
-        <div className="setting flex" onClick={() => setIsModalOpen(true)}>
-          <span class="material-symbols-outlined">switch_account</span>
-          <p className="switch">Switch</p>
+      <div className="user-profile flex">
+        <div className="profile-img">
+          <img src="/icons/profile.svg" alt="" />
+        </div>
+        <div className="user-details">
```

Comment:
I assume it should be from DB and will be handled by @Ganeshdckap

Reply:
This file is not used in our application